package.json

```diff
@@ -169,7 +169,7 @@
     "dns-sync": "^0.2.1",
     "elastic-apm-node": "^3.7.0",
     "elasticsearch": "^16.7.0",
-    "http-aws-es": "6.0.0",
+    "http-aws-es": "npm:@zhongnansu/http-aws-es@6.0.1",
```

Comment:
why do we change the version here? are we allowed to do this?

Reply:
This is a drop-in replacement as a short term solution to support legacy client  before https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2220 is completed . See this comment for the reasoning and long term solution.  https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3623#issuecomment-1513858249

I've seen this kind of drop-in being done here as short term solution. Besides, the `http-aws-es` package is only used for data source feature
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/8121c9df29db8e04d2b188ec29408c0e0ce02a6d/package.json#L462

Reply:
discussed with the team and reached an agreement to do this as a short term solution.